### PR TITLE
service_implementation.py: Add Exception Handling

### PR DESCRIPTION
**Pull request by sladyn98**
Added Exception hadling to the HTTP tests

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,13 +2,14 @@ language: python
 sudo: required
 
 python:
-  - 3.6
+  - 2.7.9
```
Comment (glaslos) [inline .travis.yml]: This tool should be tested with Python 3.6

File: .travis.yml [hunk 2]
```diff
@@ -30,4 +32,6 @@ install:
   # - docker image build --tag checkpot .
 
 script:
+  - hg clone --config ui.tls=false https://vladflorea@bitbucket.org/vladflorea/python-nmap
```
Comment (glaslos) [inline .travis.yml]: you want to keep this requirement in the requirements.txt

**Review by glaslos — CHANGES_REQUESTED** [review]

**Comment by sladyn98** [comment]
Yeah yeah i pushed these changes here by mistake  I will revert them soon
Sorry.

**Comment by sladyn98** [comment]
@glaslos Does this need any more reviews.

**Pull request closed without merging**